### ISSUE TITLE
letsencrypt container error

### ISSUE THREAD

**Issue by bereska** [BenMcH/bitwarden_rs-docker-compose#5]
thank you for the great tool)
however I have this error after running docker-compose -d up:
https://gist.github.com/bereska/7151f8dfad7594c304ab4cda25dbcbcb

In my DNS records I have my main domain parustrans.com A record pointed to one server IP 1 and bw.parustrans.com A record pointed a different server IP 2.
Ports 80 and 443 are open.
What could cause a problem here?

thank you

**Comment by bereska** [comment]
pls ignore, my bad, i didn't know aws inbound rules override my ufw rules, so after opening port 80 in aws console it worked like a charm, now i have my portable versatile password manager running!
I have a question though. Is it possible to disable anyone registration? make it invite only?

**Comment by BenMcH** [comment]
Yes! I just looked at the upstream project, [dani-garcia/bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) and it looks like setting the environment variable `SIGNUPS_ALLOWED` to false will disable registration. This should just need to be added to the docker-compose file as an environment variable and to the .env template. I'd gladly accept a PR for this as long as we keep signups enabled by default
